estimagic/inference/bootstrap.py

```diff
@@ -0,0 +1,97 @@
+import pandas as pd
+
+from estimagic.inference.bootstrap_ci import _check_inputs
+from estimagic.inference.bootstrap_ci import _concatenate_functions
+from estimagic.inference.bootstrap_ci import compute_ci
+from estimagic.inference.bootstrap_estimates import _mean
+from estimagic.inference.bootstrap_estimates import get_bootstrap_estimates
+from estimagic.inference.bootstrap_samples import get_seeds
+
+
+def bootstrap(
+    data,
+    f=_mean,
+    n_draws=1000,
+    cluster_by=None,
+    ci_method="percentile",
+    alpha=0.05,
+    seeds=None,
+    return_seeds=False,
```

Comment:
Related to the general comment regarding `get_seeds`, I would not return them.

Reply:
Done in #202